python_base/Dockerfile

```diff
@@ -19,42 +19,40 @@
 # In applying this licence, CERN does not waive the privileges and immunities
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
-FROM python:3.5
+FROM centos
 
-RUN apt-get update --fix-missing && \
-    apt-get install -y \
-      python-virtualenv \
-      python-dev \
-      git \
-      nodejs \
-      npm \
-      libpq-dev \
-      build-essential \
-      libxml2-dev \
-      libxslt1-dev \
-      libjpeg-dev \
-      libfreetype6-dev \
-      libtiff-dev \
-      libmagickwand-dev \
-      imagemagick \
-      ghostscript \
-      xpdf \
-      libffi-dev \
-      liblzma-dev && \
-    ln -s /usr/bin/nodejs /usr/bin/node && \
-    npm install -g \
-      npm \
-      node-sass \
-      clean-css \
-      requirejs \
-      uglify-js
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
Maybe the image will get even smaller if these RUN commands are joined into a single one.